src/Repositories/CourseRepository.php

```diff
@@ -80,6 +80,13 @@ public function queryAll(): Builder
             ->leftJoin('categories', 'categories.id', '=', 'category_course.category_id');
     }
 
+    /**
+     * @param array $search
+     * @param int|null $skip
+     * @param int|null $limit
```

Comment:
This comment gives us no new information, as all these params already have type hints